Ideology/DefInjected/ThoughtDef/Precepts_Cannibalism.xml

```diff
@@ -0,0 +1,215 @@
+  <!-- EN: human leather {0} (+2) -->
+  <HumanLeatherApparel_Horrible.stages.human_leather_2.label>{0} из человеческой кожи (+2)</HumanLeatherApparel_Horrible.stages.human_leather_2.label>
+  <!-- EN: Why am I wearing so much human leather? This is horrible! -->
+  <HumanLeatherApparel_Horrible.stages.human_leather_2.description>Почему на мне столько одежды из человеческой кожи? Это ужасно!</HumanLeatherApparel_Horrible.stages.human_leather_2.description>
+  <!-- EN: human leather {0} etc -->
+  <HumanLeatherApparel_Horrible.stages.human_leather_etc.label>{0} из человеческой кожи (+X)</HumanLeatherApparel_Horrible.stages.human_leather_etc.label>
+  <!-- EN: I look like a freak, this is horrifying! -->
+  <HumanLeatherApparel_Horrible.stages.human_leather_etc.description>Я выгляжу как {PAWN_gender ? отбитый : отбитая} {PAWN_gender ? психопат : психопатка}! Это ужасно!</HumanLeatherApparel_Horrible.stages.human_leather_etc.description>
+  
+  <!-- EN: human leather {0} -->
+  <HumanLeatherApparel_Preferred.stages.human_leather.label>{0} из человеческой кожи</HumanLeatherApparel_Preferred.stages.human_leather.label>
+  <!-- EN: I feel good wearing human leather apparel. -->
+  <HumanLeatherApparel_Preferred.stages.human_leather.description>Мне нравится моя одежда из человеческой кожи.</HumanLeatherApparel_Preferred.stages.human_leather.description>
+  <!-- EN: human leather {0} (+1) -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_1.label>{0} из человеческой кожи (+1)</HumanLeatherApparel_Preferred.stages.human_leather_1.label>
+  <!-- EN: I enjoy wearing the skin of others. -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_1.description>Мне нравится носить кожу других людей.</HumanLeatherApparel_Preferred.stages.human_leather_1.description>
+  <!-- EN: human leather {0} (+2) -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_2.label>{0} из человеческой кожи (+2)</HumanLeatherApparel_Preferred.stages.human_leather_2.label>
+  <!-- EN: I wear the faces of my enemies. This makes me happy. -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_2.description>Я надел на себя одежду из лиц моих врагов. Это классно.</HumanLeatherApparel_Preferred.stages.human_leather_2.description>
+  <!-- EN: human leather {0} etc -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_etc.label>{0} из человеческой кожи (+X)</HumanLeatherApparel_Preferred.stages.human_leather_etc.label>
+  <!-- EN: I look like a real cannibal. Who could doubt me? -->
+  <HumanLeatherApparel_Preferred.stages.human_leather_etc.description>Я выгляжу как настоящий каннибал. А были какие-то сомнения?</HumanLeatherApparel_Preferred.stages.human_leather_etc.description>
+  
+  <!-- EN: human leather {0} -->
+  <HumanLeatherApparel_RequiredRavenous.stages.human_leather.label>{0} из человеческой кожи</HumanLeatherApparel_RequiredRavenous.stages.human_leather.label>
```

Comment:
Аналогично, надо переиспользовать:
```
I enjoy wearing the suffering of others.
Обожаю носить одежду, созданную из чужих страданий.

This will scare them. All will know my power.
Эта одежда — свидетельство моей силы! Все будут меня бояться!

The faces on my clothes aren't smiling, but they make *me* happy.
Их лица на моей одежде исполнены страданий... Это так приятно!

The flesh! The flesh signals my triumph!
Плоть! Лишь плоть не врёт! Моя одежда — символ моего могущества!
```